docusaurus/docs/guides/deployments/30-kubernetes/kubernetes-console.mdx

```diff
@@ -1,9 +1,48 @@
 ---
 sidebar_position: 30
 sidebar_label: Console
-title: Install the Console in Kubernetes
+title: Kubernetes Console
 ---
 
-import ConsoleHelmChartReadme from '/_remotes/helm-charts/charts/ziti-console/README.md';
+Use the console by navigating to the controller's address with path `/zac/` in a web browser. If you published the controller at `ziti.example.com:443`, then the console URL is `https://ziti.example.com/zac/`.
 
-<ConsoleHelmChartReadme/>
\ No newline at end of file
+View the console URL in the Helm release notes.
+
+```text title="Substitute your Helm release name for 'ziti-controller'"
+helm get notes ziti-controller
+```
+
+```buttonless title="Output"
+NOTES:
+
+Your release ziti-controller was upgraded.
+
+
+You have chart version 1.0.10 and app version 1.1.3.
+
+To learn more about the release, try:
+
+  $ helm status ziti-controller -n miniziti
+  $ helm get all ziti-controller -n miniziti
+
+This deployment provides an OpenZiti controller to manage an OpenZiti network.
+
+Visit the console in a web browser: https://ziti.example.com:443/zac/
+```
+
+```text title="Print the username and password"
+kubectl get secrets ziti-controller-admin-secret \
+--output go-template=\
+'{{range $k,$v := .data}}'\
+'{{printf "%s: " $k}}{{if not $v}}{{$v}}{{else}}{{$v|base64decode}}{{end}}{{"\n"}}'\
+'{{end}}'
+```
+
+```buttonless title="Output"
+admin-password: UJ2Z4xK1OpagXG94GkAXkR0M4MRNvcOh
+admin-user: admin
+```
+
+:::note
+The management API URL is the same as the client API URL unless you deployed the controller with `managementApi.service.enabled`, which "splits" the management API to have a separate URL.
```

Comment:
this note feels like it should be at the top not the bottom. overall i'm not sure what this page is really adding. doesn't seem to have loads of value as it is?